lib/peer.ts

```diff
@@ -28,6 +28,7 @@ class PeerOptions implements PeerJSOption {
 	config?: any;
 	secure?: boolean;
 	pingInterval?: number;
+	protocol?: string | string[];
```

Comment:
But then this would force the hands of users to either (1) use protcols named `x-${Lowercase<string>}`, which would not always be possible, since the user might be using a WebSocket server not controlled by then, and that WebSocket server might have a protocol name that doesn't confine to the above types...